stubs/inertia/resources/js/Pages/Teams/DeleteTeamForm.vue

```diff
@@ -81,7 +77,7 @@
 
             deleteTeam() {
                 this.form.delete(route('teams.destroy', this.team), {
-                    preserveScroll: true
+                    errorBag: 'deleteTeam'
```

Comment:
I removed `preserveScroll` here on purpose, because from my testing it actually didn't make sense (due to the redirect that happens to another page)